Sources/DistributedCluster/Cluster/SWIM/ClusterMembership+Converters.swift

```diff
@@ -29,7 +29,11 @@ extension ClusterMembership.Node {
     }
 
     func swimShell(_ system: ClusterSystem) -> SWIMActor {
-        try! SWIMActor.resolve(id: ._swim(on: self.asClusterNode!), using: system) // TODO: the ! is not so nice
+        do {
+            return try SWIMActor.resolve(id: ._swim(on: self.asClusterNode!), using: system)
+        } catch {
+            fatalError("Failed to resolve \(ActorID._swim(on: self.asClusterNode!).detailedDescription): \(error), tree: \n\(system._treeString())")
```

Comment:
Just improving error message, should not happen in real code but helps debug when we mess up assignID ;)